README.md

```diff
@@ -594,6 +594,20 @@ second entry, the error structure should look like this:
 }
 ```
 
+Additionally, you can assign errors to a field using your own functions.
```

Comment:
"using your own functions" is a bit vague. "Additionally you can also assign errors to a field that are managed outside of mstform".